package.json

```diff
@@ -7,8 +7,8 @@
     "JSON2": "0.1.0",
     "batch": "0.5.0",
     "bouncy": "3.2.2",
-    "browserify": "6.3.3",
-    "browserify-istanbul": "0.1.3",
+    "browserify": "8.1.3",
```

Comment:
Gotcha. Ok. We will just make a note of it in the change log.

On Thursday, February 19, 2015, Vincent Voyer <notifications@github.com>
wrote:

> In package.json
> <https://github.com/defunctzombie/zuul/pull/150#discussion_r25003220>:
>
> > @@ -7,8 +7,8 @@
> >      "JSON2": "0.1.0",
> >      "batch": "0.5.0",
> >      "bouncy": "3.2.2",
> > -    "browserify": "6.3.3",
> > -    "browserify-istanbul": "0.1.3",
> > +    "browserify": "8.1.3",
>
> I just did not wanted to lose time with outdated dependencies while using
> watchify. I know you do not like to update deps for the sake of updating,
> but waiting too much might do more harm than good?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/defunctzombie/zuul/pull/150/files#r25003220>.
>